58/yashaslokesh/transcripts.py

```diff
@@ -0,0 +1,64 @@
+import requests
+import json
+import base64 as b64
+import sqlite3
+import os
+import itertools as it
+
+# Don't run this script too much or you get rate-limit blocked :(
+
+FIRST_SET_TOTAL_TRANSCRIPTS = 79
+SECOND_SET_START = 96
+SECOND_SET_END = 187
+
+"""
+    Generator function that will yield the transcript number and transcript for every
+    available transcript. This is done so that the strings don't take up too much memory
+    all at once and slow down the script (subsequent operation of adding text to sqlite)
+"""
+def transcripts(url):
+    for num in it.chain(range(0, FIRST_SET_TOTAL_TRANSCRIPTS + 1), 
+                        range(SECOND_SET_START, SECOND_SET_END + 1)):
+        # Creates file names going 000.txt, ...014.txt, ... 186.txt
+        transcript_num = f'{str(num).zfill(3)}.txt'
```

Comment:
```python
>>> num = 1
>>> f'{num:0>3}'
'001'
```

Reply:
I didn't know you could do this, is this mentioned somewhere in the python docs?

Reply:
https://docs.python.org/3/library/string.html#format-specification-mini-language

Reply:
@42B thanks for reviewing, how did you land on this PR? Just curious